packages/formatter-html/src/configs/third-party-service-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "axe": {
+    "logo": {
+      "name": "axe",
+      "url": "/images/scan/axe.png",
```

Comment:
If possible, we should replace this with a SVG.

Reply:
I don't know, it is the current file used in the website.
@ststimac is it possible to have a svg for that logo?